src/sentry/web/frontend/setup_wizard.py

```diff
@@ -64,16 +64,19 @@ def get(self, request: HttpRequest, wizard_hash) -> HttpResponseBase:
         Redirects to organization whenever cache has been deleted
         """
         context = {"hash": wizard_hash, "enableProjectSelection": False}
-        key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
+        cache_key = f"{SETUP_WIZARD_CACHE_KEY}{wizard_hash}"
+
+        org_slug = request.GET.get("org_slug", None)
+        project_slug = request.GET.get("project_slug", None)
 
-        wizard_data = default_cache.get(key)
+        wizard_data = default_cache.get(cache_key)
         if wizard_data is None:
             return self.redirect_to_org(request)
 
         member_org_ids = OrganizationMemberMapping.objects.filter(
             user_id=request.user.id
         ).values_list("organization_id", flat=True)
-        org_mappings = OrganizationMapping.objects.filter(
+        org_mappings: list[OrganizationMapping] = OrganizationMapping.objects.filter(
```

Comment:
`list[OrganizationMapping]` type is not 100% correct here 😬
`.filter()` method returns `QuerySet` object, but unless you have problems with mypy, there is no need to explicitly type this

Reply:
hehe, thought so. If I do not specify anything it is `Any` for me...

Reply:
Hmm interesting, my IDE picks it up automatically 😅

Reply:
will remove it and check my dev env 👍